gemfiles/rouge_1.gemfile

```diff
@@ -0,0 +1,62 @@
+# This file was generated by Appraisal
```

Comment:
Are these gemfiles still necessary, given that `appraisal install` is executed already?